### DISCUSSION TITLE
space_domain with normalized coordinates

### DISCUSSION THREAD

**Discussion by guimilhoratti** [lululxvi/deepxde#1452] [Q&A]
Hi all, 

I am working with normalized data and dimensionless PDE in my problem. 
One question that bothers me is about space_domain (L_min and L_max) should it be normalized too, correct? 

Lx_min, Lx_max = 0, 1
Ly_min, Ly_max =  0, 1

Kind regards,

**Comment by lululxvi** [comment]
It is recommended, but you don't have to do it.

**Comment by guimilhoratti** [comment]
thank you
